packages/big-design/src/mixins/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './display';
 export * from './margins';
 export * from './paddings';
+export * from './transitions';
```

Comment:
I don't think we'll want to expose this publicly for now. After you remove it and update the imports, we'll be good to go!

Reply:
hopefully is all good now